op-chain-ops/deployer/integration_test/apply_test.go

```diff
@@ -323,3 +323,62 @@ func TestApplyExistingOPCM(t *testing.T) {
 
 	validateOPChainDeployment(t, ctx, l1Client, st)
 }
+
+func TestL2BlockTimeOverride(t *testing.T) {
+	kurtosisutil.Test(t)
+
+	lgr := testlog.Logger(t, slog.LevelDebug)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	enclaveCtx := kurtosisutil.StartEnclave(t, ctx, lgr, "github.com/ethpandaops/ethereum-package", TestParams)
+
+	service, err := enclaveCtx.GetServiceContext("el-1-geth-lighthouse")
+	require.NoError(t, err)
+
+	ip := service.GetMaybePublicIPAddress()
+	ports := service.GetPublicPorts()
+	rpcURL := fmt.Sprintf("http://%s:%d", ip, ports["rpc"].GetNumber())
+	l1Client, err := ethclient.Dial(rpcURL)
```

Comment:
optional nit: I think this functionality can be pulled out into it's own function:

```go
func setupL1Client(service *services.ServiceContext) (*ethclient.Client, error) {
	ip := service.GetMaybePublicIPAddress()
	ports := service.GetPublicPorts()
	rpcURL := fmt.Sprintf("http://%s:%d", ip, ports["rpc"].GetNumber())
	return ethclient.Dial(rpcURL)
}
```

and then reused in `TestEndToEndApply` too.

Reply:
yea agreed, there is some deduping to do in a future PR